tests/threading/test_threads.py

```diff
@@ -0,0 +1,49 @@
+import time
+import pytest
+from threading import Thread
+
+import godot
+from godot import Vector3, SurfaceTool, Mesh, MeshInstance
+
+
+def test_simple_thread():
+
+    done = []
+
+    def target():
+        done.append([True])
+
+    t = Thread(target=target)
+    t.daemon = True
+    t.start()
+    time.sleep(0.1)
```

Comment:
We'd better use [threading.Event](https://docs.python.org/3/library/threading.html#threading.Event) to avoid random failure when the tests are run on machine under heavy load